### PR TITLE
perf(web): Convert image assets to `webp` & skip flaky tests

### PR DESCRIPTION
**Pull request by antonjoel82**
### What changed? Why was the change needed?

- Skip the flaky logout test since we're switching to Playwright soon
- Convert non-provider image assets in `public/static/images` to `webp` to massively reduce size

### Validation

- Please do a visual skim to make sure I didn't miss replacing any paths. Primarily-affected areas:
   - login
   - quickstart
   - nav bar (avatar)

### CODE REVIEW

File: apps/web/public/static/images/logo-black-white.png
Comment (BiswaViraj) [inline apps/web/public/static/images/logo-black-white.png:1]: ❓ **question:** I see this file is deleted but no replacement webp file is added.
Does it mean we are not using this image anywhere?‏

**Comment by SokratisVidros** [comment]
@antonjoel82 Shall we merge this?

File: apps/web/public/static/images/logo-black-white.png
Comment (antonjoel82) [reply apps/web/public/static/images/logo-black-white.png:1]: That's correct! Sorry for not calling that out

**Comment by antonjoel82** [comment]
> @antonjoel82 Shall we merge this?

@SokratisVidros Could I please get an approval?

**Review by BiswaViraj — APPROVED** [review]

**Pull request merged**